osmoutils/store_helper_test.go

```diff
@@ -219,8 +211,9 @@ func (s *TestSuite) TestGatherValuesFromStore() {
 
 			actualValues, err := osmoutils.GatherValuesFromStore(s.store, tc.keyStart, tc.keyEnd, tc.parseFn)
 
-			if tc.expectedErr {
+			if tc.expectedErr != nil {
 				s.Require().Error(err)
+				s.Require().Equal(tc.expectedErr, err)
```

Comment:
I believe these two can be combined via the s.Require().ErrorIs call

Reply:
Combined with `s.Require().ErrorContains(...)`!